packages/vue-components/src/__tests__/Quiz.spec.js

```diff
@@ -3,6 +3,12 @@ import Quiz from '../questions/Quiz.vue';
 import Question from '../questions/Question.vue';
 import QOption from '../questions/QOption.vue';
 
+// Function to delay test until question is rendered
+export default function quizTimeout() {
```

Comment:
I think the export default is unecessary since the function is only used in this file :)

Reply:
Good catch! I was trying to export it from another utils.js file but decided not to in the end and forgot to delete it!